Session1.Rmd

```diff
@@ -151,17 +160,13 @@ here('data')
 
 ### Download files  
 
-We still need to download data for the first part of the workshop. You can do with with the function `download.file()`. We will save it in the `data` folder, where the **raw** data should go. 
+We still need to download data for the first part of the workshop. You can do it with the function `download.file()`. We will save it in the `data` folder, where the **raw** data should go. 
 
 ```{r download-files}
 download.file('bit.ly/GeospatialGapminder', here('data','gapminder_data.csv'), mode = 'wb')
```

Comment:
Hi Alex, I tried to run this code but was not able to download the data. I got the following error message: 

"download.file('bit.ly/GeospatialGapminder', here('data','gapminder_data.csv'), mode = 'wb')
trying URL 'bit.ly/GeospatialGapminder'
Error in download.file("bit.ly/GeospatialGapminder", here("data", "gapminder_data.csv"),  : 
  cannot open URL 'bit.ly/GeospatialGapminder"

Reply:
Hi Kyri, 

Thanks for spotting this! The underlying URL, has changed, so bit.ly doesn't work anymore. I have adapted the script adding a new bitly:  `https://bit.ly/geospatial_data`